src/main/java/com/farao_community/farao/gridcapa/data_bridge/FileMetadataConfiguration.java

```diff
@@ -45,4 +47,8 @@ public String getFileRegex() {
     public String getZoneId() {
         return zoneId;
     }
+
```

Comment:
this method is never used and the recovery of the value of do-unzip from the config file  is done in two different classes this one and the FileTransferFlow class so  I think we can keep only one